getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.md

```diff
@@ -73,8 +71,8 @@ to see if sbt can run your project on the command line.
 1. In the **Tasks** field, type `~run`. The `~` causes SBT to rebuild and rerun the project
 when you save changes to a file in the project.
 1. Click **OK**.
-1. On the **Run** menu. Click **Run 'Run the program'**.
-1. In the code, change `currentYear - 1` to `currentYear - 2`
+1. On the **Run** menu, click **Run 'Run the program'**.
+1. In the code, change the `println` line to read `println(s"The youngest person is ${ages.min}")`
```

Comment:
Just wondering out loud... For someone's first foray into Scala (maybe their first foray into programming at all) I just worry about throwing them into String interpolation. I kind of like the sort of trivial arithmetic example just because literally _anyone_ would stick with it without getting confused by what `${`...`}` means.